specs/latest/2.0/index.html

```diff
@@ -981,6 +981,7 @@ <h2 class="no-toc">Table of contents</h2>
               <tr><td>SHADING_LANGUAGE_VERSION</td>
                   <td>Returns a version or release number of the form <code>WebGL&lt;space&gt;GLSL&lt;space&gt;ES&lt;space&gt;3.00&lt;space&gt;&lt;vendor-specific information&gt;</code>.</td></tr>
             </table><br>
+            <p>For <code>RED_BITS</code>, <code>GREEN_BITS</code>, <code>BLUE_BITS</code>, and <code>ALPHA_BITS</code>, if active color attachments of the draw framebuffer do not have identical formats, generates an <code>INVALID_OPERATION</code> error.</p>
```

Comment:
Should this also say something about the return value from the getParameter call (i.e., returns 0)?